assets/dev/js/editor/editor.js

```diff
@@ -16,5 +16,3 @@ export class Editor extends EditorBase {
 }
 
 window.elementor = new Editor();
```

Comment:
maybe it should also be in the loader?

Reply:
I think it should be on the editor.js lets talk about it